shopify/actions/newsletter/subscribe.ts

```diff
@@ -0,0 +1,43 @@
+import { AppContext } from "../../mod.ts";
+import { Data, Variables, query } from "../../utils/queries/subscribe.ts";
+
+type CreateCustomerProps = {
```

Comment:
Is the name right?

Reply:
I think SubscribeCustomerProps is better, CreateCustomer can be used in the futurem. thanks